This patch adds the MediaTrackSettings type, which is only available in lib dom

diff --git a/node_modules/expo-camera/build/Camera.types.d.ts b/node_modules/expo-camera/build/Camera.types.d.ts
index 01aa75f..4ef8307 100644
--- a/node_modules/expo-camera/build/Camera.types.d.ts
+++ b/node_modules/expo-camera/build/Camera.types.d.ts
@@ -131,6 +131,23 @@
     focusDistance: number;
     zoom: number;
 }>;
+export declare type MediaTrackSettings = {
+    aspectRatio?: number
+    autoGainControl?: boolean
+    channelCount?: number
+    deviceId?: string
+    echoCancellation?: boolean
+    facingMode?: string
+    frameRate?: number
+    groupId?: string
+    height?: number
+    latency?: number
+    noiseSuppression?: boolean
+    resizeMode?: string
+    sampleRate?: number
+    sampleSize?: number
+    width?: number
+}
 export declare type CapturedPicture = {
     width: number;
     height: number;
